luks-tpm2

```diff
@@ -148,7 +148,7 @@ replace_tpm_key() {
   generate_keyfile
 
   echo "Replacing LUKS key in slot $TPM_KEY_SLOT on $ROOT_DEVICE..."
-  if cryptsetup luksChangeKey $ROOT_DEVICE "$KEYFILE" --key-slot $TPM_KEY_SLOT --key-file "$original_keyfile"; then
+  if cryptsetup luksChangeKey $ROOT_DEVICE "$KEYFILE" --new-key-slot $TPM_KEY_SLOT --key-file "$original_keyfile"; then
```

Comment:
Does luksChangeKey support the `--new-key-slot` option? The man page doesn't seem to mention it.

Reply:
Sorry, I didn't notice it was another command😅